src/cmd/root.go

```diff
@@ -60,37 +60,13 @@ func init() {
 	viper.BindEnv("api-timeout", "OPSLEVEL_API_TIMEOUT")
 	viper.BindEnv("workers", "OPSLEVEL_WORKERS", "OL_WORKERS")
 	cobra.OnInitialize(func() {
-		readConfig()
 		setupLogging()
 		setupOutput()
 		setupConcurrency()
 		setupAPIToken()
 	})
 }
 
-func readConfig() {
```

Comment:
I think this function and the call to it on line 63 should be retained to a degree. We still need to do the operations on line 89, 90 afaict.